strptime.c

```diff
@@ -15,8 +15,8 @@
 
    You should have received a copy of the GNU Library General Public
    License along with the GNU C Library; see the file COPYING.LIB.  If not,
-   write to the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-   Boston, MA 02111-1307, USA.  */
+   write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
```

Comment:
Since strptime.[ch] aren't ours[1] do we think this change is OK to make? I'd probably vote yes for the same reasons I'm OKing everything else, but we may lead to future ping-pong if we ever update them in the future.
[1] These should probably be moved to src/thirdparty/strptime (or something) for that very reason. Doing the same for jeeps, however, seems wrong because we've treated it as "ours" now for about 20 years and the author and original distributions are gone. I'd argue we ARE the source for jeeps now - absolutely a derived work, needing to keep credit and license, and such, but with no worry about ever picking up an refresh from upstream.

Reply:
strptime reverted, but not moved.  future updates are already difficult because our reformatting with astyle was applied to strptime.